libr/crypto/p/crypto_bech32.c

```diff
@@ -0,0 +1,217 @@
+/* radare - BSD-3-Clause - Copyright (c) 2017, 2021, 2024 - Pieter Wuille, W0nda */
+
+#include <r_lib.h>
+#include <stdint.h>
```

Comment:
just include r_crypto.h and remove all those rlib, stdint, stdio.. because those includes are already included from rcrypto.h